ocfweb/templatetags/banner.py

```diff
@@ -0,0 +1,10 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter
+def lab_hours_day_and_holiday(hours):
+    if hours.holiday:
+        return '{hours.weekday} ({hours.holiday})'.format(hours=hours)
+    return hours.weekday
```

Comment:
Eh, why did I even create that file?
On Sep 26, 2015 12:37 AM, "Chris Kuehl" <notifications@github.com> wrote:

> In ocfweb/templatetags/banner.py
> <https://github.com/ocf/ocfweb/pull/34#discussion_r40491198>:
>
> > @@ -0,0 +1,10 @@
> > +from django import template
> > +
> > +register = template.Library()
> > +
> > +
> > +@register.filter
> > +def lab_hours_day_and_holiday(hours):
> > +    if hours.holiday:
> > +        return '{hours.weekday} ({hours.holiday})'.format(hours=hours)
> > +    return hours.weekday
>
> was this file added by accident?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ocf/ocfweb/pull/34/files#r40491198>.
>